package.json

```diff
@@ -1,24 +1,25 @@
 {
```

Comment:
Regarding bugs, I'm always in favor of using more standard forms. That sounds good.

Regarding directories, repository url and dependencies, let me know which command made the change. We can decide to keep or revert the changes depending on what's introducing them.

The keywords are a good idea. Please check with those used for the WebWW GitHub project to be certain that we're consistent.